river/stats/lossy.py

```diff
@@ -0,0 +1,119 @@
+import math
+import typing
+from operator import itemgetter
```

Comment:
I would `import operator` and do `operator.itemgetter`

Reply:
Done!